pyspeckit/cubes/SpectralCube.py

```diff
@@ -682,6 +698,8 @@ def fit_a_pixel(iixy):
                 nearest_ind = np.argmin(d+1e10*(True-self.has_fit))
                 nearest_x, nearest_y = xx.flat[nearest_ind],yy.flat[nearest_ind]
                 gg = self.parcube[:,nearest_y,nearest_x]
+            elif use_neighbor_as_guess and np.any(local_fits):
+                gg = np.mean(self.parcube[:,(ypatch+y)[local_fits],(xpatch+x)[local_fits]],axis=1)
```

Comment:
could you add a comment here about what `axis=1` is doing?  I think it's averaging over the non-parameter axis so that `gg` is an array of parameters, but I can't tell from the diff alone.

This comment implies that I should change all instances of `axis=int` to `axis=informative_variable_name`